Lib/Src/Utilities.cpp

```diff
@@ -0,0 +1,35 @@
+#include <Tkge/Utilities.hpp>
+
+#ifdef _WIN32
+#ifndef WIN32_MEAN_AND_LEAN
+#define WIN32_MEAN_AND_LEAN
+#define _AMD64_
+#endif
+#include <libloaderapi.h>
+#include <minwindef.h>
+#else // defined(_WIN32)
```

Comment:
Should probably use `#elif defined(__linux__)` here. Then the only error will be the static assertion instead of "can't find includes" etc.